src/RegistryImportFeed.php

```diff
@@ -80,7 +78,7 @@ public function getLatest()
      */
     public function setAssetHandler(GeneratedAssetHandler $handler)
     {
-        $handler->getFilesystem()->addPlugin(new ListFiles);
+        $handler->getFilesystem();
```

Comment:
```suggestion
```
This doesn't actually do anything, just remove it entirely.

Reply:
DONE